webui/src/pages/auth/index.jsx

```diff
@@ -11,33 +11,17 @@ import ActivateInvitedUserPage from "./users/create-user-with-password";
 
 const Auth = () => {
     return (
-        <Switch>
-            <Route exact path="/auth">
-                <Redirect to="/auth/credentials"/>
-            </Route>
-            <Route path="/auth/login">
-                <LoginPage/>
-            </Route>
-            <Route path="/auth/resetpassword">
-                <ResetPasswordPage/>
-            </Route>
-            <Route path="/auth/credentials">
-                <CredentialsPage/>
-            </Route>
-            <Route path="/auth/users/create">
-                <ActivateInvitedUserPage/>
-            </Route>
-            <Route path="/auth/users">
-                <UsersIndexPage/>
-            </Route>
-            <Route path="/auth/groups">
-                <GroupsIndexPage/>
-            </Route>
-            <Route path="/auth/policies">
-                <PoliciesIndexPage/>
-            </Route>
-            <Redirect to="/auth/credentials" />
-        </Switch>
+        <Routes>
+            <Route path="" element={<Navigate to="credentials"/>} />
```

Comment:
Do we need this one? we have the default as last route. Don't know the changes in the new router.

Reply:
Nope, missed it. Fixed.